R/distributions.R

```diff
@@ -2461,6 +2476,19 @@ pordinal <- function(q, eta, thres, disc = 1, family = NULL, link = "logit") {
   cblapply(q, .fun)
 }
 
+#' CDF of beta-binomial distribution
+#' @importFrom extraDistr dbbinom
+dbeta_binom <- function (x, size, mu = 1, phi = 1, log = FALSE) {
+  extraDistr::dbbinom(x, size, alpha = mu * phi, beta = (1 - mu) * phi, log)
```

Comment:
Throughout, please name all arguments following an argument that is already named, e.g. `extraDistr::dbbinom(x, size, alpha = mu * phi, beta = (1 - mu) * phi, log = log)`.

Reply:
Nice that looks much safer, thanks!